public/app/config/collections/library/searchTypes.js

```diff
@@ -0,0 +1,28 @@
+(function () {
+    'use strict';
```

Comment:
And I had clearly forgotten that this was already the case, simply a removed file ;)